### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/gnucash/gnucash/) for GnuCash/Program.

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -8714,9 +8714,8 @@ msgid "GnuCash could not find the linked document."
 msgstr "O GnuCash não conseguiu obter trava (\"lock\") para %s."
 
 #: gnucash/gnome-utils/gnc-gnome-utils.c:580
-#, fuzzy
 msgid "GnuCash could not open the linked document:"
-msgstr "O GnuCash não conseguiu obter trava (\"lock\") para %s."
+msgstr ""
```
Comment (fellen) [inline po/pt_BR.po:8718]: @farribeiro There is no need to clear msgstr. As long as fuzzy is set, msgstr is not used.

**Comment by fellen** [comment]
I got this warnings on my checks:
```
msgfmt -c --check-accelerators="_" --statistics po/pt_BR.po
po/pt_BR.po:3572: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:4036: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:4200: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:4419: msgstr has too many keyboard accelerator marks '_'
po/pt_BR.po:4571: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:4579: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:5254: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:5259: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:5264: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:5269: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:6044: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:6967: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:6975: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:8104: msgstr has too many keyboard accelerator marks '_'
po/pt_BR.po:8766: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:8872: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:9254: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:13127: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:13535: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:14100: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:14391: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:15118: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:15130: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:16389: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:16429: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:16433: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:16844: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:16860: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:16942: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17017: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17143: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17163: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17229: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17379: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17407: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:17509: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:18157: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:18161: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:18283: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:18358: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19098: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19154: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19260: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19335: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19658: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19666: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:19869: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:20480: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:21010: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:21018: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:21551: msgstr lacks the keyboard accelerator mark '_'
po/pt_BR.po:21559: msgstr lacks the keyboard accelerator mark '_'
msgfmt: found 52 fatal errors
3820 translated messages, 1258 fuzzy translations, 469 untranslated messages.

```

**Comment by fellen** [comment]
Thanks!

File: po/pt_BR.po (hunk 1 above)
Comment (farribeiro) [reply po/pt_BR.po:8718]: In the weblate I deleted the translation what was wrong and sent a suggestion

Comment (farribeiro) [reply po/pt_BR.po:8718]: I usually use the suggestion instead of applying it as it needs editing. This to be reviewed

Comment (farribeiro) [reply po/pt_BR.po:8718]: A translation tip, some entries don't have the same translation, one case is this entry

Comment (fellen) [reply po/pt_BR.po:8718]: > In the weblate I deleted the translation what was wrong and sent a suggestion, and not edit po file directly

Again something learned about how weblate works. Thanks!

**Pull request merged**